packages/es-components/src/components/containers/notification/useNotification.js

```diff
@@ -29,7 +31,15 @@ const ContentWrapper = styled.div`
 
 const NotificationContent = React.forwardRef(
   (
-    { includeIcon, isDismissable, onDismiss, children, iconName, color },
+    {
+      includeIcon,
+      isDismissable,
+      onDismiss,
+      children,
+      iconName,
+      iconColor,
+      color
+    },
     ref
```

Comment:
I wonder if it's worth moving the props destructuring down to the function body. I don't see the list growing, but I think it can be easy to miss the second ref argument necessary for `forwardRef`. Moving the destructuring down would improve that visibility.